specification/loadtestservice/LoadTestService/models.tsp

```diff
@@ -132,11 +147,37 @@ enum Interval {
   PT1H,
 }
 
+@added(APIVersions.v2023_04_01_preview)
+@doc("Test type")
+enum TestType{
+  @doc("Quick Test")
+  QuickTest,
+  @doc("URL based Test")
+  UrlBasedTest,
```

Comment:
Yes, but I think this is to differentiate between the optionalLoadConfig and its property. But since it is named as testType maybe we can discuss with Purva that either should we include more types or should we rename this?

Reply:
CurlBasedTest is more appropriate here

Reply:
CurlBasedTest instead of TestType or instead of UrlBasedTest?